doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -273,6 +273,18 @@ DRGN-23518: ACL reassembly may deadlock
 
   **Workaround:** Ensure that the value of :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` is greater than the value of :kconfig:option:`CONFIG_BT_MAX_CONN`.
 
+.. rst-class:: v2-8-0
+
+DRGN-23511: Building of multilink Bluetooth applications fails.
+  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is not explicitly set.
+
+  **Workaround:** Set the Kconfig value explicitly.
+
+.. rst-class:: v2-8-0
+
+NCSDK-29354: Writing to flash makes Bluetooth hang.
+  This is due to a deadlock between flash erase operation and Bluetooth TX work both done in the system WQ.
```

Comment:
```suggestion
DRGN-23511: Building of multilink Bluetooth applications fails
  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is not explicitly set.

  **Workaround:** Set the Kconfig value explicitly.

.. rst-class:: v2-8-0

NCSDK-29354: Writing to flash makes Bluetooth hang
  This is due to a deadlock between flash erase operation and Bluetooth TX work both done in the system WQ.
```
We do not use full stop for the first line. Any workaround to the latter one? And how about the affected platform(s), or are these valid for all?

Reply:
Not sure yet, will update accordingly when I get the answer.